src/main/java/logIn/drivers/LogInDatabaseAccess.java

```diff
@@ -0,0 +1,38 @@
+package logIn.drivers;
+
+import entities.User;
+
+import logIn.use_case.LogInDsGateway;
+
+public class LogInDatabaseAccess implements LogInDsGateway {
+
+    public LogInDatabaseAccess(){
+    }
+
+
```

Comment:
It is possible to set up your db access with the projected csv files. If you look at my sign up use case, you can see that I have the csv for user projected to have its columns like (favourites, posts, isadmin, password, email, id). You should try setting up the csv library and implementing your code in a way similar to what I have in sign up. Let me know if you need help setting up